File: src/test/java/one/edee/babylon/properties/MultilinePropertyTest.java
package one.edee.babylon.properties;

import org.junit.Test;
import java.io.InputStream;
import java.util.Properties;

import static org.junit.Assert.assertEquals;

public class MultilinePropertyTest {

    @Test
    public void whenValueOfMultilinePropertyIsLoadedFromFileThenPropertyFileLineSeparatorIsReplacedWithSystemLineSeparator() throws Exception {
        InputStream propIS = getClassPathResourceStream("multiline.properties");
        FileActiveRecord props = new PropertyFileLoader().loadProperties(propIS);

        String expected = "<h3>Nahrajte soubor v jednom z níže uvedených formátů</h3>" + System.lineSeparator() +
                "  <p>Importovaný soubor může být pouze ve formátu CSV nebo XML dle níže uvedené specifikace.</p>";

        String multilineProp = props.get("orderImport.help.content").getValue();
        // Assuming value is of type String
        String multilinePropValue = multilineProp;

        assertEquals(expected, multilinePropValue);
    }

    private InputStream getClassPathResourceStream(String resourceRelativePath) {
        return this.getClass().getResourceAsStream(resourceRelativePath);
    }
}

File: src/test/kotlin/one/edee/babylon/properties/MultilinePropertyTest.kt
package one.edee.babylon.properties

import org.junit.Test
import java.io.InputStream

class MultilinePropertyTest {

    @Test
    fun `when value of multiline property is loaded from file then property file line separator is replaced with system line separator`() {
        val propIS = getClassPathResourceStream("multiline.properties")
        val props = PropertyFileLoader().loadProperties(propIS)

        val expected = "<h3>Nahrajte soubor v jednom z níže uvedených formátů</h3>" + System.lineSeparator() +
                "  <p>Importovaný soubor může být pouze ve formátu CSV nebo XML dle níže uvedené specifikace.</p>"
        val multilineProp = props["orderImport.help.content"]
        val multilinePropValue = multilineProp!!.value
//        assertEquals(expected, multilinePropValue)

    }

    private fun getClassPathResourceStream(resourceRelativePath: String): InputStream =
        this.javaClass.getResourceAsStream(resourceRelativePath)
}
